### PR TITLE
Add BTCB as a base routing pair

### PR DESCRIPTION
**Pull request by BenLeibig**
Badger native tokens bBADGER and bDIGG are paired with BTCb via cake incentivised pools.   Currently, one can not buy these tokens with anything but BTCb without high slippage.  We also had this problem with SushiSwap and Uniswap, and know that this is because BTCb is one of the base tokens.  This PR adds it as such.  Both SUSHI and Uni have already merged similar pull requests.

[X] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-swap-interface/blob/master/CONTRIBUTING.md) first
[X ] If your PR is work in progress, open it as `draft`
[X] Before requesting a review, all the checks need to pass
[X ] Explain what your PR does

### CODE REVIEW

**Comment by BenLeibig** [comment]
Just waiting for internal review from the Badger team before requesting PCS review.

**Comment by BenLeibig** [comment]
@pancake-swap @ChefKai : this is ready for review.

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**
